src/lib/utils/global-error-handler.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+module.exports.register = (server, options, next) => {
+    server.expose({
+        handleUncaught: (err) => {
+            server.log(
+                ['error'],
+                {
+                    message: err ? err.toString() : '(empty Error)',
+                    stack: err.stack
+                }
+            );
+            process.nextTick(() => process.exit(1));
```

Comment:
something is funky with the logging service.  even if the logger has initialized, calling `server.log(...)` does _not_ log immediately.  waiting for the event loop to clear does it.  would like to figure that out, but it's non-critical